ydb/core/mind/hive/hive_impl.cpp

```diff
@@ -3013,6 +3018,17 @@ void THive::UpdatePiles() {
     Execute(CreateUpdatePiles());
 }
 
+std::optional<TActorId> THive::GetPipeToTenantHive(const TNodeInfo* nodeInfo) {
+    if (!nodeInfo || nodeInfo->ServicedDomains.size() != 1) {
+        return std::nullopt;
+    }
+    TDomainInfo* domainInfo = FindDomain(nodeInfo->ServicedDomains.front());
+    if (!domainInfo || domainInfo->HiveId == 0 == domainInfo->HiveId == TabletID()) {
```

Comment:
Тут что-то странное с условием. Видимо, имелось в виду это:
```suggestion
    if (!domainInfo || domainInfo->HiveId == 0 || domainInfo->HiveId == TabletID()) {
```